extension/js/common/platform/xss.ts

```diff
@@ -79,10 +79,14 @@ export class Xss {
       }
       if ('style' in node) {
         // mitigation rather than a fix, which will involve updating CSP, see https://github.com/FlowCrypt/flowcrypt-browser/issues/2648
-        const style = (node as Element).getAttribute('style')?.toLowerCase();
+        let style = (node as Element).getAttribute('style')?.toLowerCase();
         if (style && (style.includes('url(') || style.includes('@import'))) {
           (node as Element).removeAttribute('style'); // don't want any leaks through css url()
         }
+        if (style && (style.includes('z-index') || style.includes('position'))) {
+          style = style.replace(/z-index:.+;|position:.+;/,'');
+          (node as HTMLElement).setAttribute('style', style);
+        }
```

Comment:
Also add `background:` and `background-color:`, to make it harder to mimic our badges.